README.md

```diff
@@ -52,6 +52,12 @@ cd experiment
 bin/physlr-make physical-map lr=linkedreads ref=reference protocol=stlfr
 ```
 
+If you provide a reference genome, Physlr automatically maps the physical map to the input reference and outputs a PAF file. You can run the following command which uses the PAF file to calculate assembly-like quality metrics for the physical map.
+```
+bin/physlr-make linkedreads.physlr.physical-map.reference.n10.qpos.chain.metrics.tsv lr=linkedreads ref=reference
```

Comment:
The filename for this target is dependent on the `lr` and `ref` setting right? Might be easier to make an a 'phony' target called 'ref-metrics' or something like that in the Makefile, so a user wouldn't have to manually adjust the filename?

Reply:
Done; with the current code Physlr generates this file automatically but one can still run this (with a phony target). Please see updates.